R/civis_ml.R

```diff
@@ -693,7 +693,11 @@ civis_ml_fetch_existing <- function(model_id, run_id = NULL) {
     metrics <- tryCatch(must_fetch_output_json(outputs, "metrics.json"),
                         error = function(e) NULL)
     model_info <- must_fetch_output_json(outputs, "model_info.json")
+    # re-raise any CivisML warnings
+    tryCatch( {for (warn in model_info$warnings) cat(warn)},
```

Comment:
It might be simpler to check whether `length(model_info$warnings) > 0`?

Reply:
I'd also suggest using `warning` instead of `cat` so that the warnings can be suppressed/caught by the user.

Reply:
I think I would rather keep the python details in the warning message. It makes debugging easier, and I don't want to mislead the user into thinking that the warning is coming from the R client itself.

I'm doing a tryCatch here because I think some older versions of CivisML do not have the warnings in the metadata. I'm not 100% about that, but I think this approach is safer.

I'm using `cat` so that `\n` in the exception will be rendered as a newline, but I can try wrapping it in `warning`.

Reply:
SGTM

Reply:
Ah, actually, checking the length should work since it's a list. I'll just have to update the mock in one of the tests to return a list.